addon/components/file-dropzone/component.js

```diff
@@ -39,15 +39,15 @@ const dragListener = new DragListener();
         {{#if dropzone.supported}}
           Drag and drop images onto this area to upload them or
         {{/if}}
-        {{#file-upload name="photos"
-                      accept="image/*"
-                      multiple=true
-                      onfileadd=(action "uploadImage")}}
+        <FileUpload name="photos"
+                    accept="image/*"
```

Comment:
I fear we don't support passing HTML attributes to `<FileUpload>` yet. It doesn't seem to pass them to `<input>`. At least I don't see a `...attributes` in the template.

Reply:
...attributes are applied to the parent element unless tagName is set to a empty string.